test/shared/package.json

```diff
@@ -1,3 +1,3 @@
 {
-  "type": "commonjs"
+  "type": "module"
 }
```

Comment:
This file might be redudant now, I think I added it for mocha w/ Node's require(esm)? Might be worth a check, anyhow

Reply:
Yes, as we use vitest now they are not needed